nix/pkgs.nix

```diff
@@ -8,7 +8,7 @@ let
         [
           (import (sources.autodocodec + "/nix/overlay.nix"))
           (import (sources.validity + "/nix/overlay.nix"))
-          (import (sources.sydtest + "/nix/overlay.nix"))
+          # (import (sources.sydtest + "/nix/overlay.nix"))
```

Comment:
I'm pretty sure this could be fixed by updating `sydtest` using `niv`.
There used to be a patch in there that's now upstreamed.